### PR TITLE
Added and Removed code to remove blockers

### PR DESCRIPTION
**Pull request by AV25242**
As someone who is following the steps for the very first time,
1. Two dependencies that's absolutely needed to work with database and data frames were missing. Added the `pip install`.

2. Secondly, the two `from` statements were not needed in this lesson, infact they were causing roadblock from executing the python script. So, removing 

`from sklearn.linear_model import LinearRegression` `from sklearn.metrics import mean_squared_error`

### CODE REVIEW

**Comment by Jak-MS** [comment]
@WilliamDAssafMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/machine-learning/tutorials/python-ski-rental-linear-regression-prepare-data.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ In [part four](python-ski-rental-linear-regression-deploy-model.md), you'll lear
 
 * Part two of this tutorial assumes you have completed [part one](python-ski-rental-linear-regression.md) and its prerequisites.
 
+## Install Dependencies
+
+! pip install pandas
+! pip install pyodbc
+
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/tutorials/python-ski-rental-linear-regression-prepare-data.md]:
```suggestion

This tutorial assumes you have completed its prerequisites, including [installing the necessary Python packages](python-ski-rental-linear-regression.md#prerequisites), `pandas` and `pyodbc`.

```

**Comment by WilliamDAssafMSFT** [comment]
#sign-off
